mappings/net/minecraft/item/map/MapState.mapping

```diff
@@ -52,6 +63,52 @@ CLASS net/minecraft/class_22 net/minecraft/item/map/MapState
 		ARG 1 pos
 		ARG 2 id
 		ARG 3 type
+	METHOD method_32361 copy ()Lnet/minecraft/class_22;
+	METHOD method_32362 of (BZLnet/minecraft/class_5321;)Lnet/minecraft/class_22;
+		COMMENT Creates a new map state instance for the client.
+		COMMENT <p>
+		COMMENT The client is not aware of the coordinates of the map state so its center coordinates will always be {@code (0, 0)}.
+		ARG 0 scale
+		ARG 1 showIcons
+		ARG 2 dimension
+	METHOD method_32363 of (DDBZZLnet/minecraft/class_5321;)Lnet/minecraft/class_22;
+		COMMENT Creates a new map state instance.
+		ARG 0 xCenter
+			COMMENT the absolute center X-coordinate
+		ARG 2 zCenter
+			COMMENT the absolute center Z-coordinate
+		ARG 4 scale
+		ARG 5 showIcons
+		ARG 6 unlimitedTracking
+		ARG 7 dimension
+	METHOD method_32364 zoomOut (I)Lnet/minecraft/class_22;
+		COMMENT Creates a new map state which is a zoomed out version of the current one.
+		COMMENT <p>
+		COMMENT The scale of the new map state is {@code currentScale + zoomOutScale} and clamped between {@code 0} and {@code 4}.
+		COMMENT <p>
+		COMMENT The colors are not copied, neither are the icons.
+		ARG 1 zoomOutScale
+			COMMENT the amount to add to the scale of the map
+	METHOD method_32365 putColor (IIB)Z
+		COMMENT Sets the color at the specified coordinates if the current color is different.
+		COMMENT
+		COMMENT @return {@code true} if the color has been updated, else {@code false}
+		ARG 1 x
+		ARG 2 z
+		ARG 3 color
+	METHOD method_32368 removeIcon (Ljava/lang/String;)V
+		ARG 1 id
+	METHOD method_32369 replaceIcons ([Lnet/minecraft/class_20;)V
+		ARG 1 icons
+	METHOD method_32370 setColor (IIB)V
+		ARG 1 x
+		ARG 2 z
+		ARG 3 color
+	METHOD method_32371 fromTag (Lnet/minecraft/class_2487;)Lnet/minecraft/class_22;
```

Comment:
Hmm, are we using fromTag or fromNbt nowadays?

Reply:
Looked quickly at the conventions, will rename it to `fromNbt` and rename the existing `toTag` to `toNbt`.